katcp/resource.py

```diff
@@ -595,10 +595,15 @@ def parent_name(self):
 
     @property
     def name(self):
-        """Name of this KATCP resource"""
+        """Name of this KATCPSensor"""
         return self._name
 
     @property
+    def python_id(self):
```

Comment:
A little verbose perhaps, but 'python_identifier_name' says exactly what it is. i.e. a name to use when a valid python-identifier is needed. Official python description of what is allowed:

https://docs.python.org/2/reference/lexical_analysis.html#grammar-token-identifier